### PR TITLE
added auto-sort on meetings + cleaned up some of the footnotes (bette…

### PR DESCRIPTION
**Pull request by radius314**
…r formatting) #22

### CODE REVIEW

**Comment by pjaudiomv** [comment]
nice, theres one other thing though not huge deal the, row ids aren't getting set with their ids properly because the id field in tomato is called source_id and in the virtual.json its called id. these dont mean much but just thought i should point it out

Screen Shot 2019-05-29 at 5 59 32 PM
Screen Shot 2019-05-29 at 5 59 25 PM

an easy fix would be just to rename the id in virtual server json to source_id and then make the neccesary changes in tally.js and index.html

**Comment by radius314** [comment]
This seems unrelated to sorting .  If you would like to fix it, feel free
to send a separate pull request.

**Comment by radius314** [comment]
If you are ok with this, can you approve?

**Comment by radius314** [comment]
Also the fix you mention is totally unnecessary. Not a worthy thing to fix.

**Comment by pjaudiomv** [comment]
I don't know if i did that right but i aprove

**Comment by pjaudiomv** [comment]
agree no ones taking the tally and trying to do crazy custom selective css stuff

File: js/tally.js [hunk 1]
```diff
@@ -77,6 +76,10 @@ function Tally() {
     });
 }
 
+Tally.prototype.getVirtualRootsDetailsCallback = function () {
```
Comment (jbraswell) [inline js/tally.js]: Not sure I would consider this function a callback. It’s just a sort function isn’t it? Maybe give it s more appropriate name?

**Comment by radius314** [comment]
It’s a callback from the function that gets the roots.  There could be
other things here.  Ideally I’d do this as an anonymous function, but there
are some limitations with JSONP callbacks.  So it’s aptly named.

**Comment by jbraswell** [comment]
A callback is generally a function you pass as a reference into another function. When a callback is used, there’s an implication that you can customize behavior by passing in different functions. That’s not really how this one is being used. 

This is just a regular function that’s being called by the anonymous function you’re passing in to the getJSONP function. That anonymous function is your actual callback, it just happens to have your regular sort function in the closure’s scope.

**Comment by radius314** [comment]
I do not agree with this definition.

**Comment by radius314** [comment]
If you prefer I can name ....Complete”

**Comment by jbraswell** [comment]
I’d probably just call it `sortTable` or something.

**Comment by radius314** [comment]
It’s not a correct indication of my intention.

**Comment by radius314** [comment]
It’s also not sorting, it’s rendering.

**Comment by jbraswell** [comment]
Ohhh, I see, it’s attaching itself to the `tallyHo` element? In that case wouldn’t `renderTable` be appropriate?

**Comment by radius314** [comment]
That could be wrapped in another function, but ostensibly the function is
indicating that the virtual roots processing has completed. That could call
renderTable as well as other things in the future.  A better name could be,
tableDataLoadingCompleted.

**Comment by radius314** [comment]
pushed a change that I hope will satisfy both our requirements.

**Comment by radius314** [comment]
If we ever decide that there is a proper way to house virtual meetings in 🍅, then this callback hell code with JSONP can die a sweet death.

**Pull request merged**